src/NHSD.BuyingCatalogue.Ordering.Api/Services/CompleteOrder/CompleteOrderService.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class CompleteOrderService : ICompleteOrderService
 
         public async Task<Result> CompleteAsync(Order order)
         {
-            var result = order.Complete();
+            var result = order?.Complete();
```

Comment:
Why add the null-conditional operator (`.?`) – no changes should be required here for commencement date.

Reply:
I was getting warning during implementation but resolved with changes in `*.csproj` changes, so reverting back to `internal`